src/Intl/Transliterator/Resources/unidata/ascii_ord.php

```diff
@@ -0,0 +1,266 @@
+    "\x6D" => 109,
+    "\x6E" => 110,
+    "\x6F" => 111,
+    "\x70" => 112,
+    "\x71" => 113,
+    "\x72" => 114,
+    "\x73" => 115,
+    "\x74" => 116,
+    "\x75" => 117,
+    "\x76" => 118,
+    "\x77" => 119,
+    "\x78" => 120,
+    "\x79" => 121,
+    "\x7A" => 122,
+    "\x7B" => 123,
+    "\x7C" => 124,
+    "\x7D" => 125,
+    "\x7E" => 126,
+    "\x7F" => 127,
+    "\x80" => 128,
+    "\x81" => 129,
+    "\x82" => 130,
+    "\x83" => 131,
+    "\x84" => 132,
+    "\x85" => 133,
+    "\x86" => 134,
+    "\x87" => 135,
+    "\x88" => 136,
+    "\x89" => 137,
+    "\x8A" => 138,
+    "\x8B" => 139,
+    "\x8C" => 140,
+    "\x8D" => 141,
+    "\x8E" => 142,
+    "\x8F" => 143,
+    "\x90" => 144,
+    "\x91" => 145,
+    "\x92" => 146,
+    "\x93" => 147,
+    "\x94" => 148,
+    "\x95" => 149,
+    "\x96" => 150,
+    "\x97" => 151,
+    "\x98" => 152,
+    "\x99" => 153,
+    "\x9A" => 154,
+    "\x9B" => 155,
+    "\x9C" => 156,
+    "\x9D" => 157,
+    "\x9E" => 158,
+    "\x9F" => 159,
+    "\xA0" => 160,
+    "\xA1" => 161,
+    "\xA2" => 162,
+    "\xA3" => 163,
+    "\xA4" => 164,
+    "\xA5" => 165,
+    "\xA6" => 166,
+    "\xA7" => 167,
+    "\xA8" => 168,
+    "\xA9" => 169,
+    "\xAA" => 170,
+    "\xAB" => 171,
+    "\xAC" => 172,
+    "\xAD" => 173,
+    "\xAE" => 174,
+    "\xAF" => 175,
+    "\xB0" => 176,
+    "\xB1" => 177,
+    "\xB2" => 178,
+    "\xB3" => 179,
+    "\xB4" => 180,
+    "\xB5" => 181,
+    "\xB6" => 182,
+    "\xB7" => 183,
+    "\xB8" => 184,
+    "\xB9" => 185,
+    "\xBA" => 186,
+    "\xBB" => 187,
+    "\xBC" => 188,
+    "\xBD" => 189,
+    "\xBE" => 190,
+    "\xBF" => 191,
+    "\xC0" => 192,
+    "\xC1" => 193,
+    "\xC2" => 194,
+    "\xC3" => 195,
+    "\xC4" => 196,
+    "\xC5" => 197,
+    "\xC6" => 198,
+    "\xC7" => 199,
+    "\xC8" => 200,
+    "\xC9" => 201,
+    "\xCA" => 202,
+    "\xCB" => 203,
+    "\xCC" => 204,
+    "\xCD" => 205,
+    "\xCE" => 206,
+    "\xCF" => 207,
+    "\xD0" => 208,
+    "\xD1" => 209,
+    "\xD2" => 210,
+    "\xD3" => 211,
+    "\xD4" => 212,
+    "\xD5" => 213,
+    "\xD6" => 214,
+    "\xD7" => 215,
+    "\xD8" => 216,
+    "\xD9" => 217,
+    "\xDA" => 218,
+    "\xDB" => 219,
+    "\xDC" => 220,
+    "\xDD" => 221,
+    "\xDE" => 222,
+    "\xDF" => 223,
+    "\xE0" => 224,
+    "\xE1" => 225,
+    "\xE2" => 226,
+    "\xE3" => 227,
+    "\xE4" => 228,
+    "\xE5" => 229,
+    "\xE6" => 230,
+    "\xE7" => 231,
+    "\xE8" => 232,
+    "\xE9" => 233,
+    "\xEA" => 234,
+    "\xEB" => 235,
+    "\xEC" => 236,
+    "\xED" => 237,
+    "\xEE" => 238,
+    "\xEF" => 239,
+    "\xF0" => 240,
+    "\xF1" => 241,
+    "\xF2" => 242,
+    "\xF3" => 243,
+    "\xF4" => 244,
+    "\xF5" => 245,
+    "\xF6" => 246,
+    "\xF7" => 247,
+    "\xF8" => 248,
+    "\xF9" => 249,
+    "\xFA" => 250,
+    "\xFB" => 251,
+    "\xFC" => 252,
+    "\xFD" => 253,
+    "\xFE" => 254,
+    "\xFF" => 255,
+);
+
+$result = &$data;
+unset($data);
+
+return $result;
```

Comment:
I'm curious what does this static+reference+unset trick does? what problem does it solve?

```php
static $data = array(
  // ...
);
$result = &$data;
unset($data);

return $result;
```

Reply:
@glensc It is used in this way for all symfony polyfills, so I think it's something related to PHP OPcache and old PHP versions? Maybe @nicolas-grekas knows why it's written this way?

Reply:
this used to be more performant under php5, i'm fine removing now